.travis.yml

```diff
@@ -0,0 +1,34 @@
+dist: trusty
```

Comment:
What do you think about darwin from the perspective of contributor?
Do you want to also test on darwin? Or it is enough to catch platform-dependent issues on Jenkins?